maxim/tests/test_together.py

```
@@ -29,26 +29,37 @@ def setUp(self):
-        self.assertIsNotNone(res.choices[0].message.content)
+        try:
+            res: ChatCompletionResponse = self.client.chat.completions.create(
+                model="meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo",
+                messages=[{"role": "user", "content": "What are the top 3 things to do in New York?"}],
+                max_tokens=150,
+                temperature=0.7,
+                extra_headers={
+                    "x-maxim-generation-name": "nyc_recommendations",
+                    "x-maxim-generation-tags": {
+                        "test_type": "non_streaming",
+                        "topic": "travel"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)